composeApp/src/commonMain/kotlin/com/github/familyvault/ui/components/FamilyGroupEntry.kt

```diff
@@ -79,9 +83,33 @@ fun FamilyGroupEntry(
                 }
             )
             Column {
-                Paragraph(familyGroup.name)
+                if (username != null) {
+                    Row(verticalAlignment = Alignment.CenterVertically) {
+                        Paragraph(
+                            text = familyGroup.name,
+                            maxLines = 1,
+                            overflow = TextOverflow.Ellipsis
+                        )
+                        ParagraphMuted(
+                            text = stringResource(Res.string.as_with_spaces) + username,
+                            maxLines = 1,
+                            overflow = TextOverflow.Ellipsis
+                        )
+                    }
+                } else {
+                    Paragraph(
+                        text = familyGroup.name,
+                        maxLines = 1,
+                        overflow = TextOverflow.Ellipsis
+                    )
+                }
```

Comment:
Ten kod można skrócić:
```suggestion
Row(verticalAlignment = Alignment.CenterVertically) {
    Paragraph(text = familyGroup.name, maxLines = 1, overflow = TextOverflow.Ellipsis)
    username?.let {
        ParagraphMuted(
            text = stringResource(Res.string.as_with_spaces) + username,
            maxLines = 1,
            overflow = TextOverflow.Ellipsis
        )
    }
}

```